pkg/services/auth/idimpl/service.go

```diff
@@ -151,6 +155,8 @@ func (s *Service) setUserClaims(ctx context.Context, identifier string, claims *
 	}
 
 	claims.AuthenticatedBy = info.AuthModule
+	claims.Email = ident.GetEmail()
```

Comment:
I think these two lines should come before line #147, because "Grafana-local" users don't have an AuthInfo entry therefore `GetAuthInfo` returns with an error then the function returns with `nil` without setting the Email and EmailVerified fields.

Reply:
Great catch!